crates/libcontainer/src/container/builder_impl.rs

```diff
@@ -85,9 +91,38 @@ impl ContainerBuilderImpl {
 
     fn run_container(&mut self) -> Result<Pid, LibcontainerError> {
         let linux = self.spec.linux().as_ref().ok_or(MissingSpecError::Linux)?;
-        let cgroups_path = utils::get_cgroup_path(linux.cgroups_path(), &self.container_id);
+        let base_cgroups_path = utils::get_cgroup_path(linux.cgroups_path(), &self.container_id);
+        let mut final_cgroups_path = base_cgroups_path;
+
+        if let Some(sub_cgroup_path) = &self.sub_cgroup_path {
+            if sub_cgroup_path != "/" {
+                let potential_path = final_cgroups_path.join(sub_cgroup_path);
+                let normalized =
```

Comment:
Can we use the `normalize` function?

https://github.com/youki-dev/youki/blob/eaf8ca518a2faf60fd4ccf8cee9c9f4f0605e38e/crates/libcontainer/src/utils.rs#L94